.drone.yml

```diff
@@ -10,13 +10,54 @@ steps:
     commands:
       - ./gradlew assemble --no-daemon
 
+  - name: sonar scanner
+    image: quay.io/ukhomeofficedigital/sonar-scanner:v3.0.3
+    depends_on:
+      - build project
+
   - name: test project
     image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
     commands:
       - ./gradlew check --no-daemon
     depends_on:
       - build project
 
+  - name: build & push
+    image: plugins/docker
+    settings:
+      registry: quay.io
+      repo: quay.io/ukhomeofficedigital/hocs-case-creator
+      tags:
+        - build_${DRONE_BUILD_NUMBER}
+        - ${DRONE_COMMIT_SHA}
+        - branch-${DRONE_COMMIT_BRANCH/\//_}
+    environment:
+      DOCKER_PASSWORD:
+        from_secret: QUAY_ROBOT_TOKEN
+      DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
```

Comment:
Is there a repo for this on Quay?

Reply:
Quay repo created: https://quay.io/repository/ukhomeofficedigital/hocs-case-creator?namespace=ukhomeofficedigital

ACP-7101 support ticket to allow robot token to write and read to the Quay repo.